src/south/MQTT/MQTT.class.js

```diff
@@ -1,27 +1,30 @@
 const mqtt = require('mqtt')
-const mqttWildcard = require('mqtt-wildcard')
-const { vsprintf } = require('sprintf-js')
 const { DateTime } = require('luxon')
 
-const ProtocolHandler = require('../ProtocolHandler.class')
+const SouthConnector = require('../SouthConnector.class')
+const { formatValue } = require('./utils')
 
-class MQTT extends ProtocolHandler {
+/**
+ * Class MQTT - Subscribe to data topic from a MQTT broker
+ */
+class MQTT extends SouthConnector {
   static category = 'IoT'
 
   /**
    * Constructor for MQTT
    * @constructor
-   * @param {Object} dataSource - The data source
-   * @param {Engine} engine - The engine
+   * @param {Object} settings - The South connector settings
+   * @param {Engine} engine - The Engine
    * @return {void}
    */
-  constructor(dataSource, engine) {
-    super(dataSource, engine, {
+  constructor(settings, engine) {
+    super(settings, engine, {
       supportListen: true,
       supportLastPoint: false,
       supportFile: false,
       supportHistory: false,
     })
+    this.handlesPoints = true
```

Comment:
It is not truly a point handling south, because it has no `lastPointQuery` method to be used in HistoryQuery.